# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdrkit/compile-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./wodim/CMakeLists.txt	(revision 837)
+++ ./wodim/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (CDRECORD C)
-INCLUDE_DIRECTORIES(../include ../libedc ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../libedc ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 INCLUDE(../include/AddScgBits.cmake)
 include(../include/AddSchilyBits.cmake)
 
--- ./libparanoia/CMakeLists.txt	(revision 837)
+++ ./libparanoia/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBparanoia C)
-INCLUDE_DIRECTORIES(../include ${CMAKE_BINARY_DIR} ../wodim)
+INCLUDE_DIRECTORIES(../include ${CMAKE_BINARY_DIR} ../wodim ${CMAKE_BINARY_DIR}/include)
 ADD_DEFINITIONS(-DHAVE_CONFIG_H)
 SET(LIBparanoia_SRCS gap.c isort.c overlap.c p_block.c paranoia.c pmalloc.c)
 
--- ./libhfs_iso/CMakeLists.txt	(revision 837)
+++ ./libhfs_iso/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBhfs_iso C)
-INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 ADD_DEFINITIONS(-DUSE_LARGEFILES -DABORT_DEEP_ISO_ONLY -DAPPLE_HYB -DUDF -DDVD_VIDEO -DSORTING -DHAVE_CONFIG_H -DUSE_LIBSCHILY -DUSE_SCG)
 
 
--- ./libedc/CMakeLists.txt	(revision 837)
+++ ./libedc/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBEDC C)
-INCLUDE_DIRECTORIES(../include . ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include . ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 
 AUX_SOURCE_DIRECTORY(. LIBEDC_SRCS)
 
--- ./genisoimage/CMakeLists.txt	(revision 837)
+++ ./genisoimage/CMakeLists.txt	(working copy)
@@ -3,7 +3,7 @@
 INCLUDE(../include/AddScgBits.cmake)
 INCLUDE(../include/AddSchilyBits.cmake)
 
-INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 
 INCLUDE(CheckIncludeFiles)
 CHECK_INCLUDE_FILES("magic.h" USE_MAGIC)
--- ./libusal/CMakeLists.txt	(revision 837)
+++ ./libusal/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBSCG C)
-INCLUDE_DIRECTORIES(../include ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 include(../include/AddScgBits.cmake)
 ADD_DEFINITIONS(-DUSE_RCMD_RSH)
 
--- ./icedax/CMakeLists.txt	(revision 837)
+++ ./icedax/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (icedax C)
-INCLUDE_DIRECTORIES(../include ../wodim ../libparanoia ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../wodim ../libparanoia ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 include(../include/AddScgBits.cmake)
 include(../include/AddSchilyBits.cmake)
 include(../include/AddNetworkBits.cmake)
--- ./librols/CMakeLists.txt	(revision 837)
+++ ./librols/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBROLS C)
-INCLUDE_DIRECTORIES(../include ../libhfs_iso ${CMAKE_BINARY_DIR} ../wodim)
+INCLUDE_DIRECTORIES(../include ../libhfs_iso ${CMAKE_BINARY_DIR} ../wodim ${CMAKE_BINARY_DIR}/include)
 
 SET(LIBROLS_SRCS  astoi.c astoll.c astoull.c breakline.c cmpbytes.c comerr.c error.c fexec.c fillbytes.c getargs.c getav0.c getdomainname.c geterrno.c gethostid.c gethostname.c getpagesize.c handlecond.c movebytes.c raisecond.c rename.c saveargs.c seterrno.c spawn.c strcatl.c strdup.c streql.c swabbytes.c usleep.c stdio/cvmod.c stdio/dat.c stdio/fcons.c stdio/fdown.c stdio/fdup.c stdio/ffileread.c stdio/ffilewrite.c stdio/fgetline.c stdio/file_raise.c stdio/fileclose.c stdio/fileluopen.c stdio/filemopen.c stdio/fileopen.c  stdio/filepos.c stdio/fileread.c stdio/filereopen.c stdio/fileseek.c stdio/filesize.c stdio/filestat.c stdio/filewrite.c stdio/flag.c stdio/flush.c stdio/fpipe.c stdio/niread.c stdio/niwrite.c stdio/nixread.c stdio/nixwrite.c stdio/openfd.c stdio/peekc.c default.c)
 
--- ./readom/CMakeLists.txt	(revision 837)
+++ ./readom/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (READECD C)
-INCLUDE_DIRECTORIES(../include ../wodim ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../wodim ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 INCLUDE(../include/AddScgBits.cmake)
 INCLUDE(../include/AddSchilyBits.cmake)
 
--- ./libunls/CMakeLists.txt	(revision 837)
+++ ./libunls/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (LIBunls C)
-INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR})
+INCLUDE_DIRECTORIES(../include ../libhfs_iso ../wodim ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 ADD_DEFINITIONS(-DHAVE_CONFIG_H)
 
     IF(HAVE_ICONV_H)
--- ./netscsid/CMakeLists.txt	(revision 837)
+++ ./netscsid/CMakeLists.txt	(working copy)
@@ -1,5 +1,5 @@
 PROJECT (netscsid C)
-INCLUDE_DIRECTORIES(../include ../wodim ${CMAKE_BINARY_DIR} )
+INCLUDE_DIRECTORIES(../include ../wodim ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/include)
 INCLUDE(../include/AddScgBits.cmake)
 INCLUDE(../include/AddSchilyBits.cmake)
 INCLUDE(../include/AddNetworkBits.cmake)

